unit_tests_spec.rb

```diff
@@ -0,0 +1,126 @@
+require_relative 'book'
```

Comment:
- [ ] great job with the test. please ensure that All test files should be placed in a different directory (e.g /spec) . that is There should be one file per class.
eg for testing person class . it should be inside 
`spec/person_spec.rb`